gio/src/initable.rs

```diff
@@ -18,6 +23,21 @@ impl Initable {
         unsafe { object.init(cancellable)? };
         Ok(object)
     }
+
+    // rustdoc-stripper-ignore-next
+    /// Create a new instance of an initable object of the given type with the given properties.
+    ///
+    /// Similar to [`Object::with_type`] but can fail either because the object
+    /// creation failed or because `Initable::init` failed.
+    pub fn with_type<P: IsA<Cancellable>>(
+        type_: glib::Type,
+        properties: &[(&str, &dyn ToValue)],
+        cancellable: Option<&P>,
```

Comment:
```suggestion
    pub fn with_type(
        type_: glib::Type,
        properties: &[(&str, &dyn ToValue)],
        cancellable: Option<&impl IsA<Cancellable>>,
```
We use this for auto-generated code nowadays